lib/client/src/client.rs

```diff
@@ -788,6 +898,142 @@ impl MangoClient {
         Ok(ixs)
     }
 
+    async fn deactivate_first_active_unused_serum3_orders(
+        &self,
+        account: &mut MangoAccountValue,
+    ) -> anyhow::Result<PreparedInstructions> {
+        let mut serum3_closable_order_market_index = None;
+
+        for p in account.all_serum3_orders() {
+            let open_orders_acc = self
+                .account_fetcher
+                .fetch_raw_account(&p.open_orders)
+                .await?;
+            let open_orders_bytes = open_orders_acc.data();
+            let open_orders_data: &serum_dex::state::OpenOrders = bytemuck::from_bytes(
+                &open_orders_bytes[5..5 + std::mem::size_of::<serum_dex::state::OpenOrders>()],
+            );
```

Comment:
now that this is done twice (also in serum3_cancel_all_orders), could you make a helper that goes from pubkey to open orders data?